lib/plugins/interactiveCli/index.js

```diff
@@ -40,4 +35,20 @@ module.exports = class InteractiveCli {
       },
     };
   }
+  asyncInit() {
+    if (!process.stdin.isTTY) return Promise.resove();
```

Comment:
It's a sync function, so let's not use promises.

I know the intention of `asyncInit` is to expose an _async_ setup here. Still engine that resolves it, just support promises as return values, but doesn't really require it. So we're totally safe writing sync function, and to avoid unnecessary complexity best to follow that when there's nothing _async_ we're after.

It'll be also good to add a comment on top of this function explaining why this is propagated to async init.